hamid914-shahrekord.md

```diff
@@ -0,0 +1,5 @@
+## Shahrekord, Chaharmahal-O-Bakhtiari
+
+### Places to eat
```

Comment:
@crichID Thanks 😄 I'm super excited being in the class! I recommend [Dizi](https://en.wikipedia.org/wiki/Abgoosht) in http://www.mrdizi.com) 👍 It is "must try" one